pkgs/development/libraries/libcint/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, openblas
+, python
+, pythonPackages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libcint";
+  version = "3.0.19";
+
+  src = fetchFromGitHub {
+    owner = "sunqm";
+    repo = "libcint";
+    rev = "v${version}";
+    sha256 = "0x613f2hiqi2vbhp20fcl7rhxb07f2714lplzd0vkvv07phagip9";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ openblas ];
+  cmakeFlags = [
+    "-DENABLE_TEST=1"
+    "-DQUICK_TEST=1"
+    "-DCMAKE_INSTALL_PREFIX=" # ends up double-adding /nix/store/... prefix, this avoids issue
+  ];
+
+  doCheck = true;
+  checkInputs = [ python pythonPackages.numpy ];
+  checkPhase = ''
+    make test
+  '';
+
+  meta = {
+    description = ''
+      general GTO integrals for quantum chemistry
+    '';
+    longDescription = ''
+      libcint is an open source library for analytical Gaussian integrals.
+      It provides C/Fortran API to evaluate one-electron / two-electron
+      integrals for Cartesian / real-spheric / spinor Gaussian type functions.
+    '';
+    homepage = http://wiki.sunqm.net/libcint;
+    downloadPage = https://github.com/sunqm/libcint;
+    license = lib.licenses.bsd2;
+    maintainer = with lib.maintainers; [];
```

Comment:
please add yourself as a maintainer. 

If you're not already in maintainers.nix, please add yourself in a separate commit, then amend the others. similar to https://github.com/NixOS/nixpkgs/pull/78311